apps/relying-party-func/src/functions/CreateOIDCClientConfig.ts

```diff
@@ -0,0 +1,12 @@
+import { app, HttpResponseInit } from "@azure/functions";
+
+export const healthcheckHandler = async (): Promise<HttpResponseInit> => ({
+  status: 200,
+  body: "Function up and running",
+});
+
+app.http("httpTrigger1", {
```

Comment:
```suggestion
app.http("Health", {
```
make sure all the functions have a proper name

(what is the route of this function?)

Reply:
I have updated the path 😃 , sorry what do you mean by "what is the route of the function?"?

Reply:
All Functions that are part of an Azure Function App have a `Name` and a `route` (`api/v1/foo/bar`)

![image](https://github.com/pagopa/io-fims/assets/4357400/c59bc2fc-1bec-4004-906b-e9905c60ca73)

With 

```ts
app.http("foo", {
  methods: ["GET"]
  handler: ....
})
```

You are creating a Function named `foo` with `/foo` as route. Since it is complex at scale to give a unique name that acts well as `name` (easy to understand) and `route` (REST-compliant) I suggest you to give each function a dedicated `Name` and `route`, writing:

```ts
app.http("foo", {
  methods: ["GET"],
  route: "bar",
  handler: ....
})
```

Reply:
Ok thanks, take a look [here](https://github.com/pagopa/io-fims/pull/18/commits/7265575d2eafb38eb84419da58cbdc2678bc7468)